manifests/deployments/sciety-labs/sciety-labs--prod.yaml

```diff
@@ -93,6 +92,8 @@ spec:
         - name: opensearch-secret-volume
           mountPath: /root/.secrets/opensearch/
           readOnly: true
+        - name: cache-storage
+          mountPath: /app/sciety-labs/.cache
```

Comment:
I think this is the right path - @de-code could you confirm?

Reply:
Looks correct to me